e2e-tests/demand-backup-cloud/run

```diff
@@ -200,6 +200,12 @@ main() {
 		check_backup_deletion "https://storage.googleapis.com/${backup_dest_gcp}.sst_info/sst_info.00000000000000000000" "gcp-cs"
 		check_backup_deletion "https://engk8soperators.blob.core.windows.net/${backup_dest_azure}.sst_info/sst_info.00000000000000000000" "azure-blob"
 
+		if [ "$EKS" = 1 ]; then
+			backup_name_aws_iam="on-demand-backup-aws-s3-iam"
+			desc "Run backup ${backup_name_aws_iam} for $cluster cluster using IAM"
+			run_backup_with_delete "${backup_name_aws_iam}"
+		fi
+
```

Comment:
Why don't we check backup existence in storage?

Reply:
Added thanks for pointing out :)